app/locale.js

```diff
@@ -0,0 +1,181 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const L20n = require('l20n')
+const path = require('path')
+const ipcMain = require('electron').ipcMain
+
+var menuIdentifiers = () => {
+  return [
+    'downloadsManager',
+    'confirmClearPasswords',
+    'about',
+    'quit',
+    'addToReadingList',
+    'viewPageSource',
+    'copyImageAddress',
+    'openImageInNewTab',
+    'saveImage',
+    'copyLinkAddress',
+    'openInNewSessionTab',
+    'openInNewPrivateTab',
+    'openInNewTab',
+    'openAllInTabs',
+    'disableAdBlock',
+    'disableTrackingProtection',
+    'muteTab',
+    'unmuteTab',
+    'pinTab',
+    'unpinTab',
+    'deleteFolder',
+    'deleteBookmark',
+    'editFolder',
+    'editBookmark',
+    'unmuteTabs',
+    'muteTabs',
+    'addBookmark',
+    'addFolder',
+    'newTab',
+    'closeTab',
+    'bookmarkPage',
+    'openFile',
+    'openLocation',
+    'openSearch',
+    'importFrom',
+    'closeWindow',
+    'savePageAs',
+    'spreadTheWord',
+    'share',
+    'undo',
+    'redo',
+    'cut',
+    'copy',
+    'paste',
+    'pasteWithoutFormatting',
+    'delete',
+    'selectAll',
+    'findNext',
+    'findPrevious',
+    'file',
+    'edit',
+    'view',
+    'actualSize',
+    'zoomIn',
+    'zoomOut',
+    'toolbars',
+    'reloadPage',
+    'reloadTab',
+    'cleanReload',
+    'readingView',
+    'tabManager',
+    'textEncoding',
+    'toggleDeveloperTools',
+    'toggleBrowserConsole',
+    'toggleFullScreenView',
+    'home',
+    'back',
+    'forward',
+    'reopenLastClosedWindow',
+    'showAllHistory',
+    'clearHistory',
+    'bookmarks',
+    'addToFavoritesBar',
+    'window',
+    'minimize',
+    'zoom',
+    'selectNextTab',
+    'selectPreviousTab',
+    'moveTabToNewWindow',
+    'mergeAllWindows',
+    'downloads',
+    'history',
+    'bringAllToFront',
+    'help',
+    'sendUsFeedback',
+    'services',
+    'hideOthers',
+    'showAll',
+    'newPrivateTab',
+    'newSessionTab',
+    'newWindow',
+    'reopenLastClosedTab',
+    'print',
+    'findOnPage',
+    'checkForUpdates',
+    'preferences',
+    'bookmarksManager',
+    'importBookmarks',
+    'reportAnIssue',
+    'submitFeedback',
+    'bookmarksToolbar',
+    'bravery',
+    'replaceAds',
+    'blockAds',
+    'allowAdsAndTracking',
+    'block3rdPartyCookie',
+    'blockPopups',
+    'httpsEverywhere'
+  ]
+}
```

Comment:
would be good to eventually have this figured out dynamically but the scope for that is too big for now.